<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>tiposarticulodlg</class>
 <widget class="QDialog" name="tiposarticulodlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tipos Articulo</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear" >
          <property name="toolTip" >
           <string>Nuevo Tipo Artículo</string>
          </property>
          <property name="whatsThis" >
           <string>Nuevo Tipo Artículo</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="toolTip" >
           <string>Borrar Tipo Artículo</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar Tipo Artículo</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_editDelete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_guardar" >
          <property name="toolTip" >
           <string>Guardar Tipo Artículo</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar Tipo Artículo</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_save.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>320</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_listTipos" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="mui_detalles" >
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Imprimir Pedido</string>
       </property>
       <property name="whatsThis" >
        <string>Imprimir Pedido</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../bulmages-recursos-graficos.qrc" >:/Genericos32x32/images/pendientes/i_edit.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Q3GroupBox" name="groupBox1" >
     <property name="title" >
      <string>Contenido de la familia</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="labelalgo" >
          <property name="text" >
           <string>Código</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_codTipo" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Descripción</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Q3TextEdit" name="m_descTipo" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>5</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>581</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar" >
       <property name="text" >
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar" >
       <property name="text" >
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends></extends>
   <header>q3textedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../bulmages-recursos-graficos.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_cancelar</sender>
   <signal>clicked()</signal>
   <receiver>tiposarticulodlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>537</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel" >
     <x>77</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mui_detalles</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox1</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>18</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
